frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

```diff
@@ -79,15 +79,17 @@ export default ({
                                     />
                                     &nbsp;by&nbsp;
                                     {info.owner_name ? info.owner_name : '<anonymous>'}
+                                    &nbsp;
                                     <div
                                         onClick={(ev) => {
                                             setAnchorEl(ev.currentTarget);
                                         }}
                                         className={classes.permissions}
-                                        style={{ float: 'right' }}
+                                        style={{ display: 'inline-block' }}
                                     >
                                         {renderPermissions(info)}
                                     </div>
+                                    &nbsp;
```

Comment:
It probably will be better to use {" "} for spaces, ditto for line 82, 119 and 80

Reply:
You're write. It's more consistent. Another way is to change CSS instead of adding html like this.